Xamarin.Essentials/Calendar/Calendar.android.cs

```diff
@@ -116,10 +149,12 @@ static async Task<IReadOnlyList<IEvent>> PlatformGetEventsAsync(string calendarI
                     HasAttendees = cur.GetInt(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.HasAttendeeData)) == 1,
                     HasExtendedProperties = cur.GetInt(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.HasExtendedProperties)) == 1,
                     Status = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Status))
-                });
+                };
+            }
+            else
+            {
+                throw new Exception("o oh");
```

Comment:
Do we want to throw an exception, or just return null if the event isn't found?

Reply:
Changed to return NullReferenceException to match the behaviour of iOS (and uwp in a future PR)